syntax = "proto3";

service Namer {
  rpc EnglishFullName (NameRequest) returns (NameResponse) {}
}

message NameRequest {
  string first_name = 1;
  string last_name = 2;
  string middle_name = 3;
  string prefix = 4;
  string suffix = 5;
}

message NameResponse {
  string full_name = 1;
}
